src/js/components/nft/nft-collection-view.jsx

```diff
@@ -119,61 +121,44 @@ class NftCollectionView extends Component {
         });
     }
 
-    renderNftImage({
-                       imageProps,
-                       photo
-                   }) {
-        const {
-                  src,
-                  alt,
-                  srcSet,
-                  sizes,
-                  ...restImageProps
-              } = imageProps;
-        return (<Card className={'nft-card'}>
-            <Card.Img variant="top" src={src} alt={alt} {...(srcSet ? {
-                srcSet,
-                sizes
-            } : null)} {...restImageProps} style={{maxWidth: 800}}/>
-            <Card.Body>
-                <Card.Title style={{
-                    width   : `calc(${sizes} - 4vw)`,
-                    maxWidth: 720
-                }}>{photo.hash}</Card.Title>
-                <div className={'nft-value-container card-text'}>
-                    {svg.millix_logo()}
-                    <div className={'millix-value'}>
-                        <span>{format.millix(photo.amount, false)}</span>
-                    </div>
-                </div>
-                <div style={{
-                    display      : 'flex',
-                    flex         : 1,
-                    flexDirection: 'row'
-                }}>
-                    <Button variant="primary" onClick={restImageProps.onClick}>transfer</Button>
-                    <div style={{
-                        flex         : 1,
-                        display      : 'flex',
-                        flexDirection: 'row-reverse'
-                    }}>
-                        <Button
-                            className="icon_only"
-                            variant="outline-secondary"
-                            onClick={() => this.setState({
-                                modal_show_burn_confirmation: true,
-                                nft_selected                : photo
-                            })}>
-                            <FontAwesomeIcon
-                                icon={'chain-slash'}/>
-                        </Button>
-                    </div>
-                </div>
-            </Card.Body>
-        </Card>);
+    renderNftImage(nft_list) {
+        let nft_list_row = [];
```

Comment:
`nft_list_row` doesn't make sense, change it to `nft_list` ( in all the places you use this name